src/utils/daos_hdlr.c

```diff
@@ -1535,6 +1536,741 @@ cont_destroy_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+static int
+parse_filename(const char* path, char** _obj_name, char** _cont_name)
+{
+	char *f1 = NULL;
+	char *f2 = NULL;
+	char *fname = NULL;
+	char *cont_name = NULL;
+	int rc = 0;
+	if (path == NULL || _obj_name == NULL || _cont_name == NULL)
+			return -EINVAL;
+
+	if (strcmp(path, "/") == 0) {
+		*_cont_name = strdup("/");
+		if (*_cont_name == NULL)
+			return -ENOMEM;
+		*_obj_name = NULL;
+		return 0;
+	}
+	f1 = strdup(path);
+	if (f1 == NULL) {
+		rc = -ENOMEM;
+		goto out;
+	}
+
+	f2 = strdup(path);
+	if (f2 == NULL) {
+		rc = -ENOMEM;
+		goto out;
+	}
+	fname = basename(f1);
+	cont_name = dirname(f2);
+
+	if (cont_name[0] == '.' || cont_name[0] != '/') {
```

Comment:
```suggestion
	if (cont_name[0] != '/') {
```
`== '.'` is a subset of `!= '/'`, so checking for `'.'` is redundant

(I've also seen this function in other places, not including this PR)